app/views/reservations/_reservation_fields.html.haml

```diff
@@ -1,8 +1,15 @@
 .datetime-block
-  = f.input :reserve_start_date, :input_html => { :class => 'datepicker' }
+  - start_disabled = f.object.persisted? && !f.object.reserve_start_at_editable?
+  = f.input :reserve_start_date, :input_html => { :class => 'datepicker', disabled: start_disabled }
   .control-group
     .controls
       .string.optional.control-label &nbsp;
-      = time_select f, :reserve_start, minute_step: @instrument.reserve_interval
+      = time_select f, :reserve_start, { minute_step: @instrument.reserve_interval }, disabled: start_disabled
+      - if start_disabled
```

Comment:
Is this really necessary? I would expect it to simply not update the start at field if it doesn't get the parameters.